components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -48,11 +45,21 @@ private const val DEFAULT_SERVER_URL = "https://crash-reports.mozilla.com/submit
 class MozillaSocorroService(
     private val applicationContext: Context,
     private val appName: String,
-    private val serverUrl: String = DEFAULT_SERVER_URL
+    private val appId: String = BuildConfig.MOZ_APP_ID,
```

Comment:
nit: Let's use the same default as GeckoView for the app id... then it will just magically work again for all apps, without them needing to do any change.

Reply:
Understood